*** Starting uWSGI 2.0.17.1 (64bit) on [Tue Apr 30 19:30:15 2019] ***
compiled with version: 5.4.0 20160609 on 07 February 2019 15:01:38
os: Linux-4.4.0-31-generic #50-Ubuntu SMP Wed Jul 13 00:07:12 UTC 2016
nodename: ubuntu
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 1
current working directory: /home/python/test/bishe
writing pidfile to uwsgi2.pid
detected binary path: /home/python/.virtualenvs/django_py3/bin/uwsgi
chdir() to /home/python/test/bishe/dailyfresh
your processes number limit is 11807
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to TCP address 127.0.0.1:8081 fd 3
Python version: 3.5.2 (default, Nov 12 2018, 13:43:14)  [GCC 5.4.0 20160609]
Set PythonHome to /home/python/.virtualenvs/django_py3
Python main interpreter initialized at 0x25ab440
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 416880 bytes (407 KB) for 8 cores
*** Operational MODE: preforking+threaded ***
failed to open python file dailyfresh/wsgi.py
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 12850)
spawned uWSGI worker 1 (pid: 12851, cores: 2)
spawned uWSGI worker 2 (pid: 12852, cores: 2)
spawned uWSGI worker 3 (pid: 12853, cores: 2)
spawned uWSGI worker 4 (pid: 12854, cores: 2)
SIGINT/SIGQUIT received...killing workers...
worker 1 buried after 1 seconds
worker 2 buried after 1 seconds
worker 3 buried after 1 seconds
worker 4 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.17.1 (64bit) on [Tue Apr 30 19:37:34 2019] ***
compiled with version: 5.4.0 20160609 on 07 February 2019 15:01:38
os: Linux-4.4.0-31-generic #50-Ubuntu SMP Wed Jul 13 00:07:12 UTC 2016
nodename: ubuntu
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 1
current working directory: /home/python/test/bishe
writing pidfile to uwsgi2.pid
detected binary path: /home/python/.virtualenvs/django_py3/bin/uwsgi
chdir() to /home/python/test/bishe/dailyfresh
your processes number limit is 11807
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to TCP address 127.0.0.1:8081 fd 3
Python version: 3.5.2 (default, Nov 12 2018, 13:43:14)  [GCC 5.4.0 20160609]
Set PythonHome to /home/python/.virtualenvs/django_py3
Python main interpreter initialized at 0x1fdd450
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 416880 bytes (407 KB) for 8 cores
*** Operational MODE: preforking+threaded ***
failed to open python file dailyfresh/wsgi.py
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 12962)
spawned uWSGI worker 1 (pid: 12963, cores: 2)
spawned uWSGI worker 2 (pid: 12964, cores: 2)
spawned uWSGI worker 3 (pid: 12965, cores: 2)
spawned uWSGI worker 4 (pid: 12966, cores: 2)
--- no python application found, check your startup logs for errors ---
[pid: 12963|app: -1|req: -1/1] 127.0.0.1 () {44 vars in 756 bytes} [Tue Apr 30 19:37:44 2019] GET /favicon.ico => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 12963|app: -1|req: -1/2] 127.0.0.1 () {40 vars in 764 bytes} [Tue Apr 30 19:37:50 2019] GET /index => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 12963|app: -1|req: -1/3] 127.0.0.1 () {40 vars in 703 bytes} [Tue Apr 30 19:45:52 2019] GET /favicon.ico => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 12963|app: -1|req: -1/4] 127.0.0.1 () {40 vars in 708 bytes} [Tue Apr 30 19:46:04 2019] GET /favicon.ico => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 12963|app: -1|req: -1/5] 127.0.0.1 () {44 vars in 756 bytes} [Tue Apr 30 20:06:15 2019] GET /favicon.ico => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 12963|app: -1|req: -1/6] 127.0.0.1 () {40 vars in 638 bytes} [Tue Apr 30 20:41:09 2019] GET /favicon.ico => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 12963|app: -1|req: -1/7] 127.0.0.1 () {42 vars in 736 bytes} [Tue Apr 30 20:41:22 2019] GET /goods/1 => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 12963|app: -1|req: -1/8] 127.0.0.1 () {40 vars in 638 bytes} [Tue Apr 30 20:41:25 2019] GET /favicon.ico => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
SIGINT/SIGQUIT received...killing workers...
worker 1 buried after 1 seconds
worker 2 buried after 1 seconds
worker 3 buried after 1 seconds
worker 4 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.17.1 (64bit) on [Tue Apr 30 20:46:54 2019] ***
compiled with version: 5.4.0 20160609 on 07 February 2019 15:01:38
os: Linux-4.4.0-31-generic #50-Ubuntu SMP Wed Jul 13 00:07:12 UTC 2016
nodename: ubuntu
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 1
current working directory: /home/python/test/bishe
writing pidfile to uwsgi2.pid
detected binary path: /home/python/.virtualenvs/django_py3/bin/uwsgi
chdir() to /home/python/test/bishe
your processes number limit is 11807
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to TCP address 127.0.0.1:8081 fd 3
Python version: 3.5.2 (default, Nov 12 2018, 13:43:14)  [GCC 5.4.0 20160609]
Set PythonHome to /home/python/.virtualenvs/django_py3
Python main interpreter initialized at 0x11b8450
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 416880 bytes (407 KB) for 8 cores
*** Operational MODE: preforking+threaded ***
/home/python/.virtualenvs/django_py3/lib/python3.5/site-packages/haystack/utils/__init__.py:12: RemovedInDjango19Warning: django.utils.importlib will be removed in Django 1.9.
  from django.utils import importlib

WSGI app 0 (mountpoint='') ready in 1 seconds on interpreter 0x11b8450 pid: 13996 (default app)
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 13996)
spawned uWSGI worker 1 (pid: 13998, cores: 2)
spawned uWSGI worker 2 (pid: 13999, cores: 2)
spawned uWSGI worker 3 (pid: 14000, cores: 2)
spawned uWSGI worker 4 (pid: 14001, cores: 2)
/home/python/.virtualenvs/django_py3/lib/python3.5/site-packages/haystack/backends/whoosh_cn_backend.py:9: RemovedInDjango19Warning: The utilities in django.db.models.loading are deprecated in favor of the new application loading system.
  from django.db.models.loading import get_model

[pid: 13998|app: 0|req: 1/1] 127.0.0.1 () {42 vars in 736 bytes} [Tue Apr 30 20:46:59 2019] GET /goods/1 => generated 10074 bytes in 14620 msecs (HTTP/1.1 200) 4 headers in 224 bytes (1 switches on core 0)
